core/controlplane/config/templates/cloud-config-controller

```diff
@@ -2050,7 +2050,7 @@ write_files:
                     - --cloud-provider=aws
                     - --skip-nodes-with-local-storage=false
                     - --expander=least-waste
-                    - --node-group-auto-discovery=asg:tag=k8s.io/cluster-autoscaler/enabled
+                    - --node-group-auto-discovery=asg:tag=k8s.io/cluster-autoscaler/enabled,kubernetes.io/cluster/{{.ClusterName}}
```

Comment:
CA with automatic node group discovery feature enabled doesn't break when there are two or more kube-aws clusters anymore 😉 See https://github.com/kubernetes/autoscaler/pull/137 for more info